tsconfig.browser.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist.browser",
+    },
+    "target": "es5",
+    "module": "es2015",
```

Comment:
This and `lib` confuses me a bit. Is this the recommended `module` for the browser?

Reply:
I'll have to investigate which options are best here. I got these values from [this blog post](https://medium.com/collaborne-engineering/typescript-create-library-for-nodejs-and-browser-fece291d517f) but unfortunately there is no explanation for them so I'll learn more about the properties and report back.